File: src/coffee/source/helloworld/HelloWorld.java
package coffee.source.helloworld;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class HelloWorld extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView text = (TextView)findViewById(R.id.my_text);
        text.setText("Hello, world!");
    }
}

File: src/coffee/source/helloworld/HelloWorld.kt
package coffee.source.helloworld

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class HelloWorld: Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val view = findViewById(R.id.my_text) as TextView
        view.text = "Hello, kotlin world!"
    }
}
